src/snowflake/cli/app/snow_connector.py

```diff
@@ -157,8 +158,6 @@ def _update_connection_application_name(connection_parameters: Dict):
     """Update version and name of app handling connection."""
     connection_application_params = {
         "application_name": PARAM_APPLICATION_NAME,
-        "internal_application_name": PARAM_INTERNAL_APPLICATION_NAME,
-        "internal_application_version": VERSION,
```

Comment:
This params were a fallback if customer opts out from telemetry.
Was it a problem?

Reply:
They cause unexpected issues on side effect. Using those parameters requires corresponding logic on server side.